### ISSUE TITLE
'pipeline' is not a valid tag library

### ISSUE THREAD

**Issue by henare** [ciudadanointeligente/write-it#1118]
On a fresh Vagrant install you get lots of these errors when running tests and you get the same exception when trying to connect to the webserver:

```
======================================================================
ERROR: test_get_absolute_url_i18n (nuntium.tests.writeitinstances_test.InstanceTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vagrant/nuntium/tests/writeitinstances_test.py", line 76, in test_get_absolute_url_i18n
    response = self.client.get(writeitinstance1.get_absolute_url())
  File "/vagrant/global_test_case.py", line 113, in get
    request = super(WriteItRequestFactory, self).get(path, data=data, secure=secure, **extra)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 473, in get
    response = super(Client, self).get(path, data=data, **extra)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 280, in get
    return self.request(**r)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/test/client.py", line 444, in request
    six.reraise(*exc_info)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 137, in get_response
    response = response.render()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/response.py", line 82, in rendered_content
    content = template.render(context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 140, in render
    return self._render(context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/test/utils.py", line 85, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 840, in render
    bit = self.render_node(node, context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/debug.py", line 78, in render_node
    return node.render(context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in render
    compiled_parent = self.get_parent(context)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 98, in get_parent
    return get_template(parent)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader.py", line 138, in get_template
    template, origin = find_template(template_name)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader.py", line 127, in find_template
    source, display_name = loader(name, dirs)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader.py", line 43, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader.py", line 49, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader.py", line 149, in get_template_from_string
    return Template(source, origin, name)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 125, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 153, in compile_string
    return parser.parse()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/loader_tags.py", line 215, in do_extends
    nodelist = parser.parse()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/base.py", line 278, in parse
    compiled_result = compile_func(self, token)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/template/defaulttags.py", line 1084, in load
    (taglib, e))
TemplateSyntaxError: 'pipeline' is not a valid tag library: ImportError raised loading pipeline.templatetags.pipeline: cannot import name setting_changed

----------------------------------------------------------------------
Ran 577 tests in 127.709s

FAILED (SKIP=13, errors=43)
Destroying test database for alias 'default'...
(writeit-virtualenv)vagrant@precise64:/vagrant$
```

**Comment by henare** [comment]
Actually this exception is also being raised when you try to connect to the web server.

Since the tests are passing on Travis I guess this is related to the Vagrant environment.

**Comment by henare** [comment]
Fixed in 1a8c7412460bb38ddd522cc29223b8682cc3f7b5 - Silly me, I just needed to update my repo :unamused:
